lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -408,19 +408,18 @@ void SessionStateEnforcer::m5g_move_to_inactive_state(
 void SessionStateEnforcer::m5g_move_to_active_state(
     std::unique_ptr<SessionState>& session, SetSmNotificationContext notif,
     SessionStateUpdateCriteria* session_uc) {
-  bool get_gnb_teid = false;
-  bool get_upf_teid = false;
-
+  const std::string upf_ip = get_upf_n3_addr();
   /* Reattach or get  rules to the session */

```

Comment:
can you fix the space between `get` and `rules` please

Reply:
Yes. Fixed it.